docs/docs/files-gatsby-looks-for-in-a-plugin.md

```diff
@@ -1,10 +1,23 @@
 ---
-title: Files Gatsby Looks for in a Plugin
+title: Creating a Generic plugin
```

Comment:
I personally think this will confuse people who aren't creating generic plugins, as it will make it harder to find this information. Is it possible to create two pages to serve different (yet overlapping) use cases?